source/PlayerInfo.cpp

```diff
@@ -2880,31 +2892,31 @@ set<Ship *> PlayerInfo::GetGroup(int group)
 
 // Keep track of any outfits that you have sold since landing. These will be
 // available to buy back until you take off.
-const map<const Outfit *, int> &PlayerInfo::GetStock() const
+const map<const Outfit *, int> &PlayerInfo::GetOutfitStock() const
 {
-	return stock;
+	return outfitStock;
 }
 
 
 
-int PlayerInfo::Stock(const Outfit *outfit) const
+int PlayerInfo::OutfitStock(const Outfit *outfit) const
```

Comment:
So you're saying something like this?
```suggestion
int PlayerInfo::Stock(outfit)(const Outfit *outfit) const
```
I'm pretty sure that would get confusing fast.

Reply:
Not at all.
I'm saying not touching this function, keeping it Stock(outfit*)
Then you make the other function
Stock(ship*)

Reply:
But how will we tell the code the difference?

Reply:
The compiler knows that the functions are different because they have different arguments. It automatically selects the correct one based on what you pass as the argument.